### DISCUSSION TITLE
Android App 3.121.4 is broken, error viewing mail body, only headers are visible

### DISCUSSION THREAD

**Discussion by fue36** [tutao/tutanota#6490] [Bugs] [answered]
Error message when trying to view the body of a email is "Some parts of the email failed to load due to a lost connection" when trying to view any email.  Issue started about a week ago.  Occurs on any network connection, wifi, or cell based.  Android version 7.1.2.  Issue started about a week or so ago on an older app version.  I cleared all app storage / cache data, no difference.  Web client (Firefox mobile) functions as expected.

**Comment by charlag** [comment, marked as answer]
It does look like 1 of the 2:

 * you have outdated webview: https://tuta.com/support/#web
 * you have some issue with cache (but you've cleared app data so it should not be the case)
